GOVERNANCE.md

```diff
@@ -23,12 +23,23 @@ project from a great one.
 
 ## Reviewers
 
-A reviewer is a core role within the project.
+A reviewer is a core maintainer within the project.
 They share in reviewing issues and pull requests and their LGTM counts towards the
 required LGTM count to merge a code change into the project.
 
 Reviewers are part of the organization but do not have write access.
-Becoming a reviewer is a core aspect in the journey to becoming a maintainer.
+Becoming a reviewer is a core aspect in the journey to becoming a committer.
+
+## Committers
```

Comment:
I think we have been using "core maintainers" to mention this role. Not sure we have to change the word.

Reply:
The intent is that after this change, any place that says "maintainers" or "core maintainers" is referring to reviewers and committers. So for places that mention a vote or discussion, all maintainers can be involved and vote, the threshold is just calculated based on the committers.